### ISSUE TITLE
Please activate "HiDraw" in Kernel-Options to support Logitech Unifying Receiver

### ISSUE THREAD

**Issue by guidol70** [cubieplayer/Cubian#34]
For using some wireless Logitech Keyboards and Mouse (or Combinations like the K400) we need the HiDraw Option enabled in the Kernel-Options.

Without HiDraw these Input-Devices dont work with Cubian :-( (they also dont work in Linaro for CubieBoard).

In Distros for the Raspberry HiDraw ist enabled and the Input-Devices do work with the RPi.

Many thanks in advance

**Comment by gaam** [comment]
I am also waiting for realization of this supplication.

**Comment by cubieplayer** [comment]
Ok, I will try to compile the driver for this hardware

**Comment by gaam** [comment]
Thanks in advance for the help,
gaam

**Comment by cubieplayer** [comment]
Hi, 
I recompiled the kernel with the driver build-in, but i don't know if it works.
[A10](https://github.com/cubieplayer/cubian-firmware/blob/a10/boot/uImage?raw=true)
[A20](https://github.com/cubieplayer/cubian-firmware/blob/a20/boot/uImage?raw=true)

**Comment by gaam** [comment]
Hi cubieplayer,
perfect job. 
I tested Cubieboard (A10) with new uImage and now i' can use wireless Logitech K400r as input device.
Best regards.
gaam

**Comment by gaam** [comment]
Hi cubieplayer,
my second notice:
with new uImage WiFi Mercury MW150US 2.0 isn't working.
gaam

**Comment by gaam** [comment]
Hi cubieplayer,
Mea Culpa. With realese 4 is all right.
Thanks.
gaam

**Comment by berturion** [comment]
Hello, I'm sorry for this noob question but I need also the support for my Logitech K400. I just installed Cubian for my A20 and after a first boot put it on the Nand. Now, in order to use my K400, what have I to do with this uImage ?
Do I need to flash it ?

**Comment by cubieplayer** [comment]
@berturion you can try cubian-update

**Comment by berturion** [comment]
Yes ! I got my k400 working after cubian update and forgot to update this
thread, thank you :-)

**Comment by Digusil** [comment]
Hi cubieplayer,

was this option deactivated in the last time? I have the kernel build "Linux Cubian 3.4.61+" and try to run a Logitech mk 330 on my Cubietruck. On my R-Pi the k330 works fine. An update with "cubian-update" didn't solve this problem.... any kind of idea?

**Comment by cubieplayer** [comment]
@Digusil 
Hi, The module `hidraw` does activated at kernel see config file
https://github.com/mmplayer/sunxi-kernel-config/blob/master/config-cubian-base-sun7i-3.4
https://github.com/mmplayer/sunxi-kernel-config/blob/master/config-cubian-base-sun4i

**Comment by jcpjunior** [comment]
hey friend, I just installed Cubian but I am facing the same issue with my K400R Keyboard, it didnt work. I verified my kernel config and I saw it is not HIDRAW enabled, installed the cubian update ran it and even up to date no signal of HIDRAW on kernel. should I point my kernel update to some specific address? tnks
